src/pages/home/report/ReportActionCompose.js

```diff
@@ -58,6 +58,7 @@ const propTypes = {
 const defaultProps = {
     comment: '',
     modal: {},
+    report: {},
```

Comment:
What is this change related to? I would expect the report to be available if we are loading a chat for one, did we encounter some error?

Reply:
@Julesssss told me this was throwing exceptions.

Reply:
I'm afraid I don't recall where this was causing a problem, but I had to add this locally to prevent a repeated error.

Reply:
Ah found it: the following error occurs when receiving an IOU Report notification:

![Screenshot 2021-04-20 at 12 02 37](https://user-images.githubusercontent.com/10736861/115385880-93362200-a1d0-11eb-9db8-89641560cc0a.png)